plugins/woocommerce-blocks/packages/components/label/index.tsx

```diff
@@ -45,7 +48,25 @@ const Label = ( {
 	if ( hasLabel && hasScreenReaderLabel && label !== screenReaderLabel ) {
 		return (
 			<Wrapper { ...wrapperProps }>
-				<span aria-hidden="true">{ label }</span>
+				{ allowHTML ? (
+					<RawHTML>
+						{ sanitizeHTML( label, {
+							tags: [
```

Comment:
Not blocking, but I am fairly sure if you leave this out, it uses our defaults.

```
const ALLOWED_TAGS = [ 'a', 'b', 'em', 'i', 'strong', 'p', 'br' ];
const ALLOWED_ATTR = [ 'target', 'href', 'rel', 'name', 'download' ];
```

I see no reason why those defaults couldn't be expanded to include span and style.

Reply:
I think maybe we didn't want to include the `style` in the past so this can break existing systems where we are not expecting customized styles.